raiden-cli/src/utils/formatting.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { RaidenChannel } from 'raiden-ts';
+import { ApiChannel } from '../types';
+
+export function transformChannelFormatSdkToApi(channel: RaidenChannel): ApiChannel {
+  return {
+    channel_identifier: channel.id ?? 0, // TODO: how can this be undefined?
+    token_network_address: channel.tokenNetwork,
+    partner_address: channel.partner,
+    token_address: channel.token,
+    balance: channel.balance.toString(),
+    total_deposit: channel.ownDeposit.toString(), // TODO: must this add the partnerDeposit?
+    state: channel.state,
+    settle_timeout: channel.settleTimeout ?? 0,
+    reveal_timeout: 0, // TODO: why not defined?
```

Comment:
As dsicussed, this comes from `config.revealTimeout`, this config isn't specific per channel, and at some point should actually be replaced by a dynamic value based on gasPrice police and blockchain congestion, not a static/constant value, and definitely not specific per channel.

Reply:
Yes. I think this is more complex than it might look on a first gaze. So since this could be critical once we join the scenario player, I opened [an issue](https://github.com/raiden-network/light-client/issues/1656) for this, that is up for discussion. I fear it is not the last endpoint in the spec where we need to do such a compatibility decision.